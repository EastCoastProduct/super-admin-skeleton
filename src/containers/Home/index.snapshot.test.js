import React from 'react';
import renderer from 'react-test-renderer';
import Home from './';

describe('Home component snapshot', () => {
  it('renders page', () => {
    const tree = renderer.create(
      <Home />
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
