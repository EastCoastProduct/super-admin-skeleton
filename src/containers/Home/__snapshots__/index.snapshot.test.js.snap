exports[`Home component snapshot renders page 1`] = `
<p>
  Home Page
</p>
`;
